Modules/Core/Transform/test/itkCompositeTransformTest.cxx

```diff
@@ -127,6 +127,33 @@ itkCompositeTransformTest(int, char *[])
   ITK_TEST_EXPECT_EQUAL(compositeTransform->GetNumberOfTransforms(), 0u);
   ITK_TEST_EXPECT_EQUAL(compositeTransform->GetNumberOfParameters(), 0u);
   ITK_TEST_EXPECT_EQUAL(compositeTransform->GetNumberOfFixedParameters(), 0u);
+  ITK_TEST_EXPECT_EQUAL(compositeTransform->GetParameters().Size(), 0u);
+  ITK_TEST_EXPECT_EQUAL(compositeTransform->GetFixedParameters().Size(), 0u);
+
+  {
+    CompositeType::InputPointType  inputPoint({ 1.1, 2.2 });
+    CompositeType::InputVectorType inputVector;
+    inputVector[0] = 9.1;
+    inputVector[1] = 8.2;
+
+    if (!testPoint(inputPoint, compositeTransform->TransformPoint(inputPoint)))
```

Comment:
LGTM!  Could consider using `ITK_TEST_EXPECT_TRUE` here for brevity.

Reply:
Thanks for the tip. When fixing a bug, and writing a test corresponding to the issue, it is always a struggle not to get carried away with other code improvements. Here I just use the same methodology in the rest of the file for consistency.